### ISSUE TITLE
Error "Unexpected token < in JSON at position 0" when uploading multiple files

### ISSUE THREAD

**Issue by SysEngDan** [CalderaWP/Caldera-Forms#3135]
Version: 1.8.4
WP: 5.1.1
PHP: 7.2

Uploading multiple files (ct 57) using the AFU(2.0). About half way (estimated) through the file upload, the error "Unexpected token < in JSON at position 0" is presented and the upload halts.

![image](https://user-images.githubusercontent.com/11705408/55243024-c9cd2880-5214-11e9-816c-179d0c7b25c5.png)

**Comment by Shelob9** [comment]
@SysEngDan Most likely a PHP error or some sort of HTML - wrapping a PHP error or maybe a notice that the endpoint is blocked by a security service - is coming back from the API instead of properly formed JSON. I'm going to close this as issue as it does not follow the issue template -- which asks for PHP errors and other vital information. Please go ahead and submit a new issue following the template, and if its a bug, not a site-specific issue, we'll take a look.
